src/main/java/core/basesyntax/strategy/StorageReportGenerateImpl.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.db.Storage;
+import java.util.Map;
+
+public class StorageReportGenerateImpl implements StorageReportGenerate {
+    public String getReport() {
+        StringBuilder report = new StringBuilder();
+        report.append("fruit,quantity").append(System.lineSeparator());
+        for (Map.Entry<String, Integer> entry : Storage.getAllFruits().entrySet()) {
```

Comment:
Be careful with using `Storage.getAllFruits()` directly as it can lead to potential issues with encapsulation. Consider adding a method in a service class that will handle the retrieval of this data.